ivy/functional/frontends/torch/linalg.py

```diff
@@ -1,5 +1,6 @@
 # local
 import math
+import torch
```

Comment:
Hey! The native torch shouldn't be imported in the frontends, if `ivy.svdvals` doesn't support this argument yet, you should first implement the new argument in the backend, add the related argument in the tests, and then use the ivy function in this frontend, thanks!